pkg/router/controller/factory/factory.go

```diff
@@ -124,6 +127,21 @@ func (factory *RouterControllerFactory) Create(plugin router.Plugin, watchNodes,
 		}, &kapi.Secret{}, secretEventQueue, factory.ResyncInterval).Run()
 	}
 
+	// Check ARP Cache and endpoint list
+	go utilwait.Forever(func() {
+		data, err := ioutil.ReadFile("/proc/sys/net/ipv4/neigh/default/gc_thresh3")
+		if err != nil {
+			glog.Warning("Error reading ARP neighbour information")
+		}
+		endpoints, _ := factory.KClient.Endpoints(factory.Namespace).List(metav1.ListOptions{})
```

Comment:
Do not ignore the error, endpoints may not be set and endpoints.Items may fail.
Throw warning and return (similar to arp cache fetch).